commerce_coordinator/apps/commercetools/views.py

```diff
@@ -82,3 +89,39 @@ def post(self, request):
         send_order_confirmation_email(lms_user_id, customer.email, canvas_entry_properties)
 
         return Response(status=status.HTTP_200_OK)
+
+class OrderSanctionedView(APIView):
+    """View to sanction an order and deactivate the lms user"""
+    permission_classes = [IsAdminUser]
+    authentication_classes = [] #TODO: Update this with OAuth authentication
```

Comment:
If you make this an empty list there are no ayth classes... and thus it cant be hit... (given its permissions_classes requirements)
```suggestion
    # authentication_classes = [] # TODO: Update this with OAuth authentication
```